Tools/MiniBrowser/gtk/main.c

```diff
@@ -384,7 +384,7 @@ static gboolean addSettingsGroupToContext(GOptionContext *context, WebKitSetting
             .arg_data = parseFeaturesOptionCallback,
             .arg_description = "FEATURE-LIST",
         },
-        G_OPTION_ENTRY_NULL
+        NULL
```

Comment:
Good catch, I see that `G_OPTION_ENTRY_NULL` exists since GLib 2.70 only. But this should be `{ NULL }` because otherwise the compiler may complain that a struct is being initialized with a scalar:

```suggestion
        { NULL }
```